packages/web/src/components/RedwoodApolloProvider.tsx

```diff
@@ -5,7 +5,9 @@ import {
   InMemoryCache,
   useQuery,
   useMutation,
+  HttpLink,
 } from '@apollo/client'
+import fetch from 'cross-fetch'
```

Comment:
I would like to avoid this since I don't want to increase the bundle size for something that's already implemented in browsers. Let me try and figure something out.

Reply:
If we merge in Tobbe's graphqlHooksProvider, we might not need to do this actually! I can just pass in a fake useMutation and useQuery in the prerender

Reply:
I take your point... I've only done this to get it working, otherwise you know, scope creep!

Reply:
I'm now shimming fetch into the node.js context, during prerender. This now isn't required anymore